packages/parser/src/validators/spec.ts

```diff
@@ -9,13 +9,14 @@ import { SwaggerSpecificationValidator } from './spec/swagger.js';
 
 /**
  * Validates either a Swagger 2.0 or OpenAPI 3.x API definition against cases that aren't covered
- * by their JSON Schema definitions.
+ * by their respective JSON Schema definitions.
  *
```

Comment:
Do we want to call out we've relaxed validation to make some errors warnings in this comment?